db/migrate/20200518190036_set_sa_group_owner_on_workspaces.rb

```diff
@@ -0,0 +1,27 @@
+# Backfilling permissions on existing workspaces not in the 'default' portal project by adding a
+# Service Account owned user group to all workspaces
+# This is in preparation of revoking all direct SA ownerships on workspaces
```

Comment:
I REALLY like how you're not revoking the direct SA ownerships until this change has gone out and we've had time to test.

Reply:
In truth, we don't have a way to do that yet (at least not programmatically), so even if I wanted to I can't.  But I agree that doing this in a multi-stage process is still better than all at once.